route/build/cpl/RunoffMod.F90

```diff
@@ -20,14 +20,14 @@ MODULE RunoffMod
 
     integer , pointer :: gindex(:)        ! global index consistent with map file
 
-    real(r8), pointer :: area(:)          ! area of catchment [m2]
+    real(r8), pointer :: area(:)          ! area of hru (AKA catchment) [m2]
     integer , pointer :: mask(:)          ! reach category. 1=non outle, 2=interior basin outlet, 3=ocean outlet
 
     real(r8), pointer :: qsur(:,:)        ! coupler importing surface forcing [mm/s]
     real(r8), pointer :: qsub(:,:)        ! coupler importing subsurface forcing [mm/s]
     real(r8), pointer :: qgwl(:,:)        ! coupler importing glacier/wetland/lake forcing [mm/s]
     real(r8), pointer :: qirrig(:)        ! coupler importing irrigation [mm/s] - negative
-    real(r8), pointer :: qirrig_actual(:) ! actual irrigation and available main channel storage [mm/s]
+    real(r8), pointer :: take(:)          ! actual water take from river reach [mm/s]
```

Comment:
I actually think the previous name qirrig_actual is a better descriptor over "take". Take could make sense if it were the sum of several fields that take water from the river.

Reply:
rename back to `qirrig_actual`